_scripts/instruction-widget/templates/install/snap.html

```diff
@@ -29,7 +29,9 @@ <h3>Snap Support</h3>
     <tt>apt</tt>, <tt>dnf</tt>, or <tt>yum</tt>, you should remove them before
     installing the Certbot snap to ensure that when you run the command
     <tt>certbot</tt> the snap is used rather than the installation from your OS
-    package manager.
+    package manager. The exact command to do this depends on your OS, but
+    common examples are <tt>sudo apt-get remove certbot</tt> or <tt>sudo yum
```

Comment:
let's also add `dnf` since we mention it right above?

Reply:
Sure. I was a little torn because while RHEL/Fedora is trying to push people towards `dnf` so new users may be familiar with `dnf` and not `yum`, I think `sudo yum remove certbot` will work everywhere that `sudo dnf remove certbot` does.

Being consistent makes sense though.